apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/exchange/ExchangeTable.java

```diff
@@ -0,0 +1,128 @@
+/*
+ *  Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.dao.state.exchange;
```

Comment:
Might be better not creating a package(exchange) for only one class ?

Reply:
There will be two classes - exchange + exchange_request, that is why I created package.